pages/model/[id].vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <Navbar ref="navbar">
+    <template v-slot:icon class="w-20">
+      <NuxtLink
+        to="/"
+        class="flex h-full w-20 items-center justify-center border-r-3 border-black bg-blue-ribbon-600"
+      >
+        <XMark class="h-10 w-10 text-white" />
+      </NuxtLink>
+    </template>
+    <template v-slot:content>
+      <div class="flex items-end gap-6 px-6">
+        <span class="align-bottom font-display text-3xl font-bold">{{
+          mockData[$route.params.id].title
```

Comment:
It might be useful to de-couple data fetching and using -- it might inform the interface you'll need once it becomes not-mock.

```
const model = mockData[$route.params.id]

// or even
const { getModel } = useModelData();
const model = getModel($route.params.id)
```

and then
```suggestion
          model.title
```